docs/config.rst

```diff
@@ -1,11 +1,496 @@
+
+Let's now look at the the ``release_beta`` flow as its defined in the internal cumulusci.yml file:
+
+.. code-block:: yaml
+
+   release_beta:
+        description: Upload and release a beta version of the metadata currently in packaging
+        steps:
+            1:
+                task: upload_beta
+                options:
+                    name: Automated beta release
+            2:
+                task: github_release
+                options:
+                    version: ^^upload_beta.version_number
+            3:
+                task: github_release_notes
+                ignore_failure: True  # Attempt to generate release notes but don't fail build
+                options:
+                    link_pr: True
+                    publish: True
+                    tag: ^^github_release.tag_name
+                    include_empty: True
+                    version_id: ^^upload_beta.version_id
+            4:
+                task: github_master_to_feature
+
+This flow references both ``version_id`` and ``version_number`` return values set on the ``upload_beta`` task.
+
+
+
+Using Tasks and Flows From a Different Project
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The dependency handling discussed above is used in a very specific context, to install dependency packages or metadata bundles in the ``dependencies`` flow which is a component of some other flows.
+It's also possible to use arbitrary tasks and flows from another project.
+To do this, the other project must be named in the ``sources`` section of cumulusci.yml:
+
+.. code-block:: yaml
+
+    sources:
+      npsp:
+        github: https://github.com/SalesforceFoundation/NPSP
+
+This says that when tasks or flows are referenced using the `npsp` namespace, CumulusCI should fetch the source from this GitHub repository.
+By default, it will fetch the most recent release, or the default branch if there are no releases.
+It's also possible to fetch a specific ``tag``:
+
+.. code-block:: yaml
+
+    sources:
+      npsp:
+        github: https://github.com/SalesforceFoundation/NPSP
+        tag: rel/3.163
+
+or a specific ``commit`` or ``branch``.
+
+Now it's possible to run a flow from NPSP:
+
+.. code-block:: console
+
+    $ cci flow run npsp:install_prod
+
+Or a task:
+
+.. code-block:: console
+
+    $ cci task run npsp:robot
+
+Or even to create a new flow which uses a flow from NPSP:
+
+.. code-block:: yaml
+
+    flows:
+      install_npsp:
+        steps:
+          1:
+            flow: npsp:install_prod
+          2:
+            flow: dev_org
+
+This flow will use NPSP's ``install_prod`` flow to install NPSP as a managed package, and then run this project's own ``dev_org`` flow.
+
+
+
+Plan Configurations
+^^^^^^^^^^^^^^^^^^^
+
+TODO: Content for configuring plans here
```

Comment:
Should we break that out into a separate doc section, since there's a lot that goes into MetaDeploy plans?

Reply:
That sounds good to me. I'll create a TODO link where this section is referenced at the top.